gradle/versions.gradle

```diff
@@ -142,5 +144,7 @@ dependencyManagement {
     dependency 'tech.pegasys.ethsigner.internal:core:0.4.0'
     dependency 'tech.pegasys.ethsigner.internal:file-based:0.4.0'
     dependency 'tech.pegasys.ethsigner.internal:signing-api:0.4.0'
+
+    dependency 'tech.pegasys.discovery:discovery:0.4.0-dev-b9433156'
```

Comment:
We need a production release, not a dev release.  Have they done production releases of this library yet?

Reply:
We started tagging every commit as a release in discovery because it doesn't change often. So that build referenced is not a snapshot and won't change.

No objections to someone going through the usual release steps if you need it to have a better looking version number though.

Reply:
I need the updates I made in order for it to work with besu so existing prod releases can't be used right now.

Reply:
I've been fairly consistent when it comes to not shipping SNAPSHOT libraries.  While dev-commit provides more repeatability it doesn't have the creator stamp of "we think it's done" that a plain old version does.